concepts/booleans/about.md

```diff
@@ -19,18 +75,43 @@ values into a boolean value.
 
 Unlike some other languages, `0` is not `false` and non-zero is not `true`.
```

Comment:
A student may wonder, what happens if a number value is used as a boolean? Is that a runtime panic?

Might be good to note here that it is a compile time error and the programmer is helpfully prevented this mistake.